desktop/core/requirements.txt

```diff
@@ -2,6 +2,6 @@ cryptography==41.0.1
 numpy==1.23.1
 pandas==1.4.2
 lxml==4.9.1
-PyYAML==5.4.1
+PyYAML==6.0.1
```

Comment:
Why is this change required?

Reply:
@ananya-agarwal remove this change from here, if you want to push this change then add a note in Hue documentation. Like for m1 and m2 you might need to upgrade the PyYAML......